TombEngine/Game/control/box.h

```diff
@@ -83,10 +83,24 @@ constexpr auto CLIP_ALL    = (CLIP_LEFT | CLIP_RIGHT | CLIP_TOP | CLIP_BOTTOM);
 
 constexpr auto CLIP_SECONDARY = 0x10;
 
+struct AITargetFlags
```

Comment:
Why it is called `AITargetFlags`? It should be called `AITargetInfo`.

Reply:
Will change.